src/scratchtocatrobat/tools/svgtopng.py

```diff
@@ -330,21 +210,18 @@ def _parse_and_rewrite_svg_file(svg_input_path, svg_output_path):
     if 'width' in root.attrib and float((root.attrib['width']).replace('px', '')) <= 0:
         root.attrib['width'] = '1'
 
-    if 'viewBox' not in root.attrib:
-        for child in root:
-            if re.search('.*}g', child.tag) != None:
-                if 'transform' in child.attrib:
-                    matrix_transform_attrib = child.attrib['transform']
-                    matrix_transform_attrib = re.sub(r"matrix\((\s?-?[0-9]+(\.[0-9]*)?,){4}", "matrix(1, 0, 0, 1,", matrix_transform_attrib)
-                    child.attrib['transform'] = matrix_transform_attrib
-                break
-    if 'viewBox' in root.attrib:
-        del root.attrib['viewBox']
-
-    for child in root:
+
+    for child in tree.iter():
+        if re.search('.*}g', child.tag) != None:
+            if 'transform' in child.attrib:
+                matrix_transform_attrib = child.attrib['transform']
+                matrix_transform_attrib = re.sub(r"matrix\((\s?-?[0-9]+(\.[0-9]*)?,){4}", "matrix(1, 0, 0, 1,", matrix_transform_attrib)
+                child.attrib['transform'] = matrix_transform_attrib
+
+    for child in tree.iter():
         if re.search('.*}text', child.tag) != None:
             child.attrib['x'] = '3'
-            child.attrib['y'] = '22'
+            child.attrib['y'] = '23'
```

Comment:
fixes minimal cutoff on top, IDK where these numbers come from but they seem to work, without them the text isn't in the right place

Reply:
with 23 we fix a minor cut off on top of the text (its now more centered)